### PR TITLE
[MRG +1] Periodic monitors docs

### PR DESCRIPTION
**Pull request by rennerocha**
It solves #134

### CODE REVIEW

**Comment by raphapassini** [comment]
LGTM

File: docs/source/monitors.rst [hunk 1]
```diff
@@ -6,6 +6,7 @@ Monitoring your jobs
 
 Monitors
 --------
+.. _`Monitors`
```
Comment (Gallaecio) [inline docs/source/monitors.rst]: This is malformed.

See the output of `make html` within the `docs` folder (be warned: you’ll also get existing issues unrelated to your changes). You first need to install the dependencies from `requiremets-docs.txt`, on the root folder.

You don’t need a mark at all, actually. [Section titles should already work as targets](https://stackoverflow.com/a/18704682/939364).

Same applies to the section below.

File: docs/source/monitors.rst [hunk 2]
```diff
@@ -144,8 +146,69 @@ Is there a **Basic Scrapy Suite** ready to use?
...
 
 .. automodule:: spidermon.contrib.scrapy.monitors
-    :members: SpiderCloseMonitorSuite 
+    :members: SpiderCloseMonitorSuite
 
+Periodic Monitors
+-----------------
+
+Sometimes we don't want to wait until the end of the spider execution to monitor
+it. For example, we may want to be notified as soon the number of errors reach
```
Comment (Gallaecio) [inline docs/source/monitors.rst]:
```suggestion
it. For example, we may want to be notified as soon the number of errors reaches
```
The subject is ‘number’, not ‘errors’.

File: docs/source/monitors.rst [hunk 3]
```diff
@@ -144,8 +146,69 @@ Is there a **Basic Scrapy Suite** ready to use?
...
+a value or close the spider if the time elapsed is greater than expected.
+
+You define your `Monitors`_ and `Monitor Suites`_ the same way as before, but
+you need to provide the time interval (in seconds) between each of the times the
+`Monitor Suites`_ is run.
+
+In the following exemple, we defined a periodic monitor suite that will be
```
Comment (Gallaecio) [inline docs/source/monitors.rst]:
```suggestion
In the following example, we defined a periodic monitor suite that will be
```

File: docs/source/monitors.rst [hunk 4]
```diff
@@ -144,8 +146,69 @@ Is there a **Basic Scrapy Suite** ready to use?
...
+
+.. code-block:: python
+
+    # myproject/monitors.py
+    from myproject.actions import CloseSpiderAction
+
+    @monitors.name('Periodic job stats monitor')
+    class PeriodicJobStatsMonitor(Monitor, StatsMonitorMixin):
+
+        @monitors.name('Maximum number of errors exceeded')
```
Comment (Gallaecio) [inline docs/source/monitors.rst]:
```suggestion
        @monitors.name('Maximum number of errors reached')
```

It seems a bit more accurate, but feel free to ignore this comment.

If you do decide to apply it, please consider applying it to all the new code, as there are a few more cases.

File: docs/source/settings.rst [hunk 5]
```diff
@@ -36,6 +36,19 @@ SPIDERMON_EXPRESSIONS_MONITOR_CLASS
 SPIDERMON_PERIODIC_MONITORS
 ---------------------------
 
+Default: ``{}``
+
+A dict containing the monitor suites that must be executed periodically as key and
+the time interval (in seconds) between the executions as value.
+
+For example, the following suite will be executed each 30 minutes:
```
Comment (Gallaecio) [inline docs/source/settings.rst]:
```suggestion
For example, the following suite will be executed every 30 minutes:
```

I feel the current wording is slightly wrong, but I’ve been unable to find a source supporting that claim, so feel free to leave it as is.

File: docs/source/monitors.rst (hunk 1 above)
Comment (rennerocha) [reply docs/source/monitors.rst]: Fixed.

**Pull request merged**